src/main/scala/uk/gov/nationalarchives/tdr/api/service/ConsignmentService.scala

```diff
@@ -23,4 +23,10 @@ class ConsignmentService(consignmentRepository: ConsignmentRepository, timeSourc
 
     consignments.map(rows => rows.headOption.map(row => Consignment(Some(row.consignmentid), row.userid, row.seriesid)))
   }
+
+  def previousUploadToConsignment(consignmentId: UUID): Future[Boolean] = {
```

Comment:
Minor thing, but I might be inclined to word this more generically, e.g. `consignmentHasFiles`, because the service doesn't need to know that you're checking a "previous" upload against a current one.

Reply:
Fair point.